webapp/src/projects.tsx

```diff
@@ -430,6 +443,21 @@ export class ProjectsCarousel extends data.Component<ProjectsCarouselProps, Proj
                 this.hasFetchErrors = true;
             } else {
                 this.prevGalleries = pxt.Util.concat(res.map(g => g.cards));
+                const shuffle = this.props.shuffle
+                if (shuffle) {
+                    // keep last one
+                    const last = this.prevGalleries.pop();
+                    // shuffle array
+                    const now = new Date(Date.now());
+                    const seed = now.toDateString();
```

Comment:
Could just do `const seed = new Date().toDateString();`

This is now time-zone and locale dependent, but likely the classroom will have matching settings.

Reply:
I take it back, it's only time-zone dependent, so we're all clear.

Reply:
should i make the change or not? :)

Reply:
change is just minor opt, I meant I take back the locale - dependent bit. just merge ;)